eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -383,6 +383,18 @@ jobs:
 
     # Only run tests if enabled
     - ${{ if eq(parameters.runTests, 'True') }}:
+      - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
```

Comment:
Please add a comment above explaining the use-case so that this doesn't accidentally get removed again.